### ISSUE TITLE
Install commands and fail

### ISSUE THREAD

**Issue by kathpanderson** [rackn/digitalrebar-deploy#419]
**Local install on a VM running Linux using run-in-system:**
*most commands pulled directly from documentation*

**git install:

    sudo apt-get install git

**clone repo:

    cd ~
    mkdir digitalrebar
    git clone https://github.com/rackn/digitalrebar-deploy digitalrebar/deploy
    ln -s digitalrebar/ digitalrebar/deploy/compose/digitalrebar
    cd digitalrebar/deploy
    echo "Checking prerequisites"
    ./run-in-system.sh --help
    echo "let's setup Digital Rebar!"

**creating dr_info: gedit
**dr_info is copied and pasted from https://github.com/rackn/digitalrebar-deploy/blob/master /.dr_info.example and fake keys/IDs put in, REBAR_KEY and REBAR_ENDPOINT added to file

**CLI set up (added in sudo):

    if [[ $(uname -s) == Darwin ]] ; then
        curl -so rebar https://s3-us-west-2.amazonaws.com/rebar-cli/rebar-darwin-amd64
    else
        curl -so rebar https://s3-us-west-2.amazonaws.com/rebar-cli/rebar-linux-amd64
    fi
    chmod +x rebar
    sudo mv rebar /usr/local/bin

**Install:

    sudo ./run-in-system.sh --deploy-admin=local --admin-ip=10.0.2.15

**Lots of babysitting

**Fails:

    TASK [FORWARDER wait until Digital Rebar service is up [1 upto 20 minutes]]
    fatal: [10.0.2.15]: FAILED! => {"changed":false, "elapsed": 1200, "failed": true, "msg": "Timeout when waiting for 10.0.2.15:3000"}

Lots of steps in the install are skipped... I'd guess about half of them. I don't know if that is supposed to be occurring or not. Skipped items are listed [here](https://docs.google.com/document/d/11ftihlq36DB1y1EwWQ14zmQVzFNmq40JqkTIs_AlVC0/edit?usp=sharing)

**Comment by galthaus** [comment]
What is 10.0.2.15?

**Comment by kathpanderson** [comment]
My VM's IP

**Comment by kathpanderson** [comment]
Tried to rerun, with one change. Used the following commands to fill in admin-IP:

    ip -4 addr
    export IPA=10.0.2.15/24

Then ran command as 

    ./run-in-system.sh --deploy-admin=local --access=HOST --admin-ip=$IPA

Install wouldn't even start. 

Next, I repeated by creating a new clone and trying to run again, following Stephen's steps from issue #420, however, it still wouldn't start.

Machine is still alive, but I began trying to install on another VM. Will post results as comment in issue #420, since I followed Stephen's steps.
